tck/src/main/java/ee/jakarta/tck/concurrent/spec/Platform/dd/DeploymentDescriptorServlet.java

```diff
@@ -155,29 +145,31 @@ public void testDeploymentDescriptorDefinesQualifiers() throws Throwable {
                     "Default managedExecutorService was not injected when no qualifiers were present."),
             () -> assertNotNull(injectedMESD,
                     "Deployment Descriptor defined managedExecutorService was not inject with valid qualifier."),
-            () -> assertNull(notInjectedMESD,
+            () -> assertThrows(Exception.class,
+                    () -> CDI.current().select(ManagedExecutorService.class, new AnnotationLiteral<CustomQualifier2>() { }, new AnnotationLiteral<InvalidQualifier3>() { }),
                     "A managedExecutorService was injected with both a valid and invalid qualifier.")
         );
         
         //TODO verify injected vs lookup services behave the same
         
-        ManagedExecutorService lookupDefMSES = InitialContext.doLookup("java:comp/DefaultManagedScheduledExecutorService");
-        ManagedExecutorService lookupMSESD = InitialContext.doLookup("java:global/concurrent/ScheduledExecutorD");
-
+        ManagedScheduledExecutorService lookupDefMSES = InitialContext.doLookup("java:comp/DefaultManagedScheduledExecutorService");
+        ManagedScheduledExecutorService lookupMSESD = InitialContext.doLookup("java:global/concurrent/ScheduledExecutorD");
+        
         assertAll("Managed Scheduled Executor Service Tests",
                 () -> assertNotNull(injectedDefMSES,
                         "Default managedScheduledExecutorService was not injected when no qualifiers were present."),
                 () -> assertNotNull(injectedMSESD,
                         "Deployment Descriptor defined managedScheduledExecutorService was not inject with valid qualifiers."),
-                () -> assertNull(notInjectedMSESD,
+                () -> assertThrows(Exception.class,
+                        () -> CDI.current().select(ManagedScheduledExecutorService.class, new AnnotationLiteral<CustomQualifier1>() { }),
                         "A managedScheduledExecutorService was injected with one of two required qualifiers.")
```

Comment:
I would expect the instance to be obtained here because it has all (1) of the requested qualifiers
```suggestion
                () -> assertNotNull(CDI.current().select(ManagedScheduledExecutorService.class, new AnnotationLiteral<CustomQualifier1>() { }),
                        "Could not inject a ManagedScheduledExecutorService with one of its two qualifiers.")
```